test/components/kubernetes/controlplane-single-node_test.go

```diff
@@ -0,0 +1,71 @@
+// Copyright 2020 The Lokomotive Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build packet baremetal
+// +build disruptivee2e
+
+package kubernetes_test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	testutil "github.com/kinvolk/lokomotive/test/components/util"
+)
+
+func Test_Controlplane_components_are_Deployments_on_single_controller_node_cluster(t *testing.T) {
+	client := testutil.CreateKubeClient(t)
+
+	for c := range components() {
+		c := c
+
+		t.Run(c, func(t *testing.T) {
+			t.Parallel()
```

Comment:
We should stop running diruptivee2e type tests in parallel. This will only make it harder for us to debug which caused race for others.

Reply:
This one is actually OK to run in parallel, it only verifies that all deployments converged. It's not disruptive, but should run as part of disruptive tests.

Reply:
Separate file is what I would go. Because diruptivee2e has potential to disrupt the tests :laughing:

Reply:
OK, done. Please have a look again.